projects/go-lib/src/lib/components/go-button/go-button.component.scss

```diff
@@ -42,15 +42,10 @@
 
 .go-button__icon {
   display: inline-flex;
-  padding-right: .5rem;
 }
 
-.go-button--icon-only {
-  padding: calc(.625rem - 1px) .625rem;
```

Comment:
We need this `.626rem` padding on the sides. It makes the icon only button oblong instead of a square:

![Screen Shot 2020-06-17 at 12 50 57 PM](https://user-images.githubusercontent.com/11966331/84926311-31c99200-b099-11ea-9929-cd7faa2b90f9.png)

Reply:
added it back